runtime/doc/usr_05.txt

```diff
@@ -250,121 +250,115 @@ The ":map" command (with no arguments) lists your current mappings.  At
 least the ones for Normal mode.  More about mappings in section |40.1|.
 
 ==============================================================================
-*05.4*	Adding a plugin					*add-plugin* *plugin*
-
-Vim's functionality can be extended by adding plugins.  A plugin is nothing
-more than a Vim script file that is loaded automatically when Vim starts.  You
-can add a plugin very easily by dropping it in your plugin directory.
-{not available when Vim was compiled without the |+eval| feature}
-
-There are two types of plugins:
-
-    global plugin: Used for all kinds of files
-  filetype plugin: Only used for a specific type of file
-
-The global plugins will be discussed first, then the filetype ones
-|add-filetype-plugin|.
-
-
-GLOBAL PLUGINS						*standard-plugin*
-
+*05.4*	Plugins					*add-plugin* *plugin*
+
+Vim's functionality can be extended by adding plugins.  A plugin is
+essentially a set of Vim script files that are loaded automatically when Vim
+starts or on some other event (like opening a file of a certain type).
+Plugins can be used to add support to more programming languages, extend Vim's
+command "vocabulary", simplify some use-cases, add some bling to Vim, and even
+turn it into a different sort of application.  Thankfully, the community has
+made available on the net a great ammount of plugins that any user can benefit
+from.
+
+You can add a plugin very easily by dropping it in your |vimfiles| directory,
+but there are more advanced ways to manage plugins, which will be described
+below.
+
+KINDS OF PLUGIN                                         *plugin-types*
+
+There are mainly two kinds of plugin: global and filetype-specific.
+
+Global plugins are sourced on startup and usually add functionality that will
+be available regardless of the opened files.  For example, the |matchparen|
```

Comment:
Maybe should read:

Global plugins are sourced on startup and usually add functionality that is available regardless of which files are open.